comments/comment.py

```diff
@@ -0,0 +1,33 @@
+'''Module representing the comments'''
+from datetime import datetime
+
+COMMENTS = []
+
+class Comment:
+    '''Defintions for the comment entity'''
+    def __init__(self, author, message):
+        self.comment_id = len(COMMENTS) + 1
+        self.author = author
+        self.message = message
+        self.created_at = str(datetime.now())
+
+    def add_comment(self, comment):
```

Comment:
I don't think it's appropriate to get the comment directly from the view. You should probably create the comment from the instance of this class. As follows.
```
def add_comment(self):
         comment = {
            "comment_id": self.comment_id,
            "author": self.author,
            "message": self.message,
            "created_at": self.created_at
         }

         COMMENTS.append(comment)
```

Reply:
I've now implemented the feature.